src/test/java/util/xcmailr/AbstractTest.java

```diff
@@ -72,25 +38,32 @@ private static void deleteTempFile(File tempFile)
             {
                 Files.delete(tempFile.toPath());
             }
-            catch (Exception e)
+            catch (final Exception e)
             {
+                // double apostrophe needed, otherwise MessageFormat.format() won't work
                 System.out.println(MessageFormat.format("Couldn''t delete temporary file: ''{0}'' caused by {1}",
-                                                        tempFile.getAbsolutePath(), e));
+                                                        tempFile.getAbsoluteFile(), e));
             }
         }
     }
 
+    protected static void writeProperties()
```

Comment:
Maybe we should rather rename this method to `savePropertiesAndApply`? What do you think?